app/src/main/kotlin/com/infinity/foodstandards/network/FoodStandardsRepo.kt

```diff
@@ -0,0 +1,62 @@
+package com.infinity.foodstandards.network
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import com.infinity.foodstandards.model.Establishment
+import com.infinity.foodstandards.model.EstablishmentResponse
+import com.infinity.foodstandards.model.LocalAuthoritiesResponse
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import retrofit2.Call
+import retrofit2.Response
+import kotlin.random.Random
+
+class FoodStandardsRepo(
+    private var api: FoodStandardsApi,
+    private val dispatcher: CoroutineDispatcher = Dispatchers.IO
+) {
+
+    // Get list of authorities from food standards API
+    suspend fun getLocalAuthorities(): Result<LocalAuthoritiesResponse> = withContext(dispatcher) {
+        try {
+            Result.success(api.getAuthorities())
+        } catch (exception: Exception) {
+            Log.e(TAG, "Exception calling getLocalAuthorities", exception)
+            Result.failure(exception)
+        }
+    }
+
+    // Get list of establishments from API
+    fun getEstablishments(id: Int): MutableLiveData<EstablishmentResponse> {
+        val apiData: MutableLiveData<EstablishmentResponse> = MutableLiveData()
+
+        val establishmentResponse = EstablishmentResponse()
```

Comment:
I think it would be worth adding in a comment here as people tend to get confused at this point.

Something like `TODO: get real data from API`

Reply:
Or we just delete this bit to stop any confusion?